youtube_dl/extractor/pbs.py

```diff
@@ -685,6 +686,17 @@ def extract_redirect_urls(info):
                         ttml_caption_suffix, '/%d_Encoded.vtt' % (ttml_caption_id + 2)),
                     'ext': 'vtt',
                 }])
+        else:
+            captions = try_get(info, lambda x: x['cc'], dict) or {}
+
+            if captions:
+                subtitles['en'] = []
+                for caption_url in captions.values():
+                    subtitles['en'].extend([{
+                        'ext': re.search(r'\.(\w{3,4})$',
```

Comment:
not need, there is generic code that will extract the subtitle extension from the subtitle URL.